lib/tools/adb-commands.js

```diff
@@ -344,6 +344,36 @@ methods.toggleGPSLocationProvider = async function (enabled) {
   await this.setSetting('secure', 'location_providers_allowed', `${enabled ? "+" : "-"}gps`);
 };
 
+/**
+ * Permit access to all non-SDK APIs, but keep warnings in the log.
+ * https://developer.android.com/preview/restrictions-non-sdk-interfaces
+ *
+ * Commands don't lead crash or something, thus simply this method calls putting all setting.
```

Comment:
I think this sentence could be safely skipped, since the link to the detailed description is already available.